index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <div class="main-mobile">
+        <header class="app-bar">
+
+            <nav class="toolbar">
+                <a class="logo-placeholder" href="#">
+                    <p id="logo-text">logo</p>
+                </a>
+                <a class="toolbar-menu" href="#">
+                    <img id="toolbar-img" src="icons/menu/Union.png" alt="menu">
+                </a>
+            </nav>
+        </header>
+
+        <div class="headline">
+            <p id="primary-text">Hey there. I'm Yuvenal</p>
+            <p id="supporting-text">I'am a Software Developer</p>
+            <p id="description">I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <div class="social-media">
+                <a href="https://github.com/yuvenalmash" id="github-icon">
+                    <img src="icons/github/Vector.png" alt="github"></a>
+                <a href="https://linkedin.com/in/yuvenal-njoroge" id="linkedin-icon">
+                    <img src="icons/linkedin/Group.png" alt="linkedin"></a>
+                <a href="#" id="angelist">
+                    <img src="icons/angelist/Vector.png" alt="angelist"></a>
+                <a href="https://twitter.com/_juvenal_" id="twitter-icon">
+                    <img src="icons/twitter/Vector.png" alt="twitter"></a>
+                <a href="https://medium.com/@yuvenal.njoroge" id="medium-icon">
+                    <img src="icons/medium/Vector.png" alt="medium"></a>
+            </div>
```

Comment:
- [ ] Kindly instead of wrapping your icons inside a `div`, use `ul` tag and `il` tags. Like this  👇️
```
        <ul class="social-media">
                <li><a href="https://github.com/yuvenalmash" id="github-icon">
                    <img src="icons/github/Vector.png" alt="github"></a></li>
                <li><a href="https://linkedin.com/in/yuvenal-njoroge" id="linkedin-icon">
                    <img src="icons/linkedin/Group.png" alt="linkedin"></a></li>
                <li><a href="#" id="angelist">
                    <img src="icons/angelist/Vector.png" alt="angelist"></a></li>
                 <li><a href="https://twitter.com/_juvenal_" id="twitter-icon">
                    <img src="icons/twitter/Vector.png" alt="twitter"></a></li>
                 <li><a href="https://medium.com/@yuvenal.njoroge" id="medium-icon">
                    <img src="icons/medium/Vector.png" alt="medium"></a></li>
            </ul>
```